file_paths.md

```diff
@@ -99,7 +97,6 @@ The files annotated by an '*' indicate that they are created by package installa
             libssl.so *
             ruby *
                 vendor_ruby *             # ruby code
-                    cfacter.rb *
                     facter.rb *
```

Comment:
Also, FYI it is a generated file, which is why you won't find it in the facter repo. However, you would find https://github.com/puppetlabs/facter/blob/8886a003ac6459e56d78cd3d2d98c6d1e8b1898b/lib/facter.rb.in